docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -152,7 +156,29 @@ To manage the media controller features in your application, you must learn to h
    mcServerInfo.sendPlaybackState('PAUSE', successCallback, errorCallback);
    ```
 
-## Receiving Notifications from the Server
+## Setting and Getting Server Icon URI
+
+You can update the icon URI address of a media controller server, which can then be accessed
+from the client application. The default value of the server icon URI attribute is null. If
+you want to know how to use the icon URI attribute in your media controller application follow
```

Comment:
Please add a comma before "follow these steps" and rephrase as:

To use the icon URI attribute in your media controller application, follow these steps:

Reply:
done